lib/grandpa/message.go

```diff
@@ -13,9 +13,20 @@ import (
 	"github.com/ChainSafe/gossamer/pkg/scale"
 )
 
+type GrandpaMessageType int64
```

Comment:
nit uint8 should be enough

```suggestion
type GrandpaMessageType uint8
```